tests/e2e/kubectl/kubectl.go

```diff
@@ -127,7 +127,7 @@ func (p JsonPatch) String() string {
 	} else {
 		value = fmt.Sprintf("\"%s\"", p.Value)
 	}
-	return fmt.Sprintf("[{\"op\": \"%s\", \"path\": \"%s\", \"value\":%s}]", p.Op, p.Path, value)
+	return fmt.Sprintf("{\"op\": \"%s\", \"path\": \"%s\", \"value\":%s}", p.Op, p.Path, value)
```

Comment:
```suggestion
	return fmt.Sprintf(`{"op": "%s", "path": "%s", "value":%s}`, p.Op, p.Path, value)
```

It become much readable with backticks